backend/audit/intakelib/checks/check_uei_schema.py

```diff
@@ -0,0 +1,30 @@
+import json
+import logging
+from jsonschema import validate
+from jsonschema.exceptions import ValidationError
+
+from django.conf import settings
+from audit.intakelib.intermediate_representation import get_range_by_name
+from audit.intakelib.common import (
+    get_message,
+    build_range_error_tuple,
+)
+
+logger = logging.getLogger(__name__)
+
+
+def verify_auditee_uei_schema(ir):
+    """Verify that the auditee UEI schema is valid."""
+    uei_range = get_range_by_name(ir, "auditee_uei")
+    uei_value = uei_range.get("values")
+    with open(f"{settings.OUTPUT_BASE_DIR}/UeiSchema.json") as file:
+        schema = json.load(file)
+    for index, uei in enumerate(uei_value):
```

Comment:
Index not used?

Reply:
I wanted to use this to identify the UEI cell but since there is only one UEI per workbook, it appeared unnecessary.